src/style.css

```diff
@@ -0,0 +1,204 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search-form {
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+}
+
+.search-form:hover,
+.search-form:focus-within {
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.search-box {
+  position: relative;
+  box-sizing: border-box;
+  height: 70px;
+
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-box:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-box:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: none;
+  border-radius: 3px;
+}
+
+.search-icon {
+  box-sizing: border-box;
+
+  width: 19px;
+  height: 19px;
+
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
+
+.div-input {
+  width: 158px;
+  height: 19px;
+
+  position: relative;
+  left: 62px;
+  top: 35.71%;
+  bottom: 37.14%;
+}
+
+.search-input {
+  position: absolute;
+  left: 0%;
+  right: 18.35%;
+  top: 0%;
+  bottom: -15.79%;
```

Comment:
It's bad practice to use absolute position for content that can be places without it. Refactor.